src/afterTownHallData.ts

```diff
@@ -68,6 +68,11 @@ const data = {
     },
   ],
   breakoutRooms: [
+    {
+      name: 'Decentralizing the Potential for Human Collaboration',
+      host: 'Jeffrey and Lewis',
+      description: 'Let's talk about the impact of Project Catalyst proposals',
```

Comment:
This quoting is breaking the code. Use double quotes (instead of single quotes) around the whole thing, since there's a single quote inside the content